examples/replicaset/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  rs-0:
```

Comment:
Can we use a better naming maybe? People will just copy paste this and then forever be stuck with rs-0 :P  `mongod-replica-0`? Same for the --replSet

Reply:
How is that better than what I did? I am happy to rename the example rs to something else but not sure replica is even better... 

Why prefix it with `mongod` its obviously a mongo replicaset you are using mongo to connect to it.

Reply:
I imagine these running in a larger environment (lets say k8s) and then you have a bunch of pods and it is very helpful to prefix them by something, so they are alphabetically ordered. Doesn't super much matter by what its prefixed.

How about this naming convention:

`mongo-${rs || config || shard-a}-${replica number}` 

replica sets get rs
config servers config
shards `shard-` letters

Reply:
This I am by the mongod was the issue didn't really see the need